.github/workflows/build-site.yml

```diff
@@ -0,0 +1,41 @@
+name: Build Website
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: cd docs-md
+    - run: npm ci
+    - run: npm install
```

Comment:
`npm ci` and `npm install` should be redundant

Reply:
👍 fixed